sweep/fee_bumper.go

```diff
@@ -141,6 +141,10 @@ type BumpRequest struct {
 	// ExtraTxOut tracks if this bump request has an optional set of extra
 	// outputs to add to the transaction.
 	ExtraTxOut fn.Option[SweepOutput]
+
+	// Immediate is an optional parameter that can be used to specify that
```

Comment:
micro nit: i wouldnt say it is optional right? the parameter is always set

Reply:
kk now updated